cmake/Hunter/config.cmake

```diff
@@ -32,6 +40,11 @@ hunter_config(
 hunter_config(
   Protobuf
   VERSION 3.14.0-4a09d77-p0
+  CMAKE_ARGS
```

Comment:
But aren't these cmake policies passed from our toolchain `cmake/toolchain/base.cmake` ?

Reply:
Oh, I didn't know we already set them globally.

I see these variable values from `base.cmake` in config.cmake if I print them with `message()`,
but I'm not sure that they are propagated to the Protobuf library.

Inside
~/.hunter/_Base/Download/Hunter/0.24.3/10738b5/Unpacked/cmake/projects/Protobuf/hunter.cmake
they call `hunter_cmake_args()` ([declared here](https://github.com/ruslo/hunter/blob/master/cmake/modules/hunter_get_cmake_args.cmake)), but it is not clear to me whether it merges with the parent scope or overrides it.

I don't know how to verify it other than see if you get these link warnings before and after this patch.
Do you have them on master and on this branch?

Reply:
Hmm, I do get linker warning w/o these explicit `CMAKE_ARGS` for protobuf. Strange, how come Hunter doesn't propagate cmake settings from the toolchain 🤷‍♂️